### PR TITLE
refactor(orc8r): Segregation of s6aproxy servicer as External

### PR DESCRIPTION
**Pull request by shivesh-wavelabs**
Signed-off-by: shivesh <shivesh.ojha@wavelabs.ai>

## Summary

This PR deals with the segregation of swx_proxy.proto external gRPC servicers implementation.
This changes are part of the ongoing work which details the distinction between Orc8r-internal gRPC endpoints vs. external, gateway-oriented gRPC endpoints.
## Test Plan

All Test Complete
All UT Complete

### CODE REVIEW

**Comment by shivesh-wavelabs** [comment]
this PR has been closed due to facing an issue at the reflecting the all changes

File: feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/relay_router.go [hunk 1]
```diff
@@ -25,7 +25,6 @@ const (
 	DiamUnableToDeliverErr = 3002
```
Comment (sabithaatla) [inline feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/relay_router.go:25]: Changes not required

File: feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/southbound/relay_router.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/*
```
Comment (sabithaatla) [inline feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/southbound/relay_router.go:1]: File has to be renamed. It should not be created as new one

File: feg/cloud/go/services/feg_relay/feg_relay/main.go [hunk 3]
```diff
@@ -55,7 +56,8 @@ func main() {
 
 	// Register Neutral Host Routing services
 	nhServicer := nh_servicers.NewRelayRouter()
-	protos.RegisterS6AProxyServer(srv.GrpcServer, nhServicer)
+	s6aproxy_servicers := s6a_proxy_servicers.NewRelayRouter()
```
Comment (sabithaatla) [inline feg/cloud/go/services/feg_relay/feg_relay/main.go:59]: Changes not required

**Pull request closed without merging**